templates/page_header/page-header.tpl.php

```diff
@@ -6,8 +6,29 @@
  */
 ?>
 <div class="ecl-page-header">
-  <?php print render($breadcrumb); ?>
+  <?php print render($header_bottom); ?>
   <div class="ecl-page-header__body">
       <?php print render($page_header); ?>
-    </div>
+      <div class="ecl-page-header__identity">
+        <?php print render($site_name); ?>
+      </div>
```

Comment:
If we follow the ECL completely, we should check if the site_name exists before rendering it.